package com.transsion.baselib.db.video;

import com.transsion.baselib.db.video.UGCCollectionVideoGroupDao;
import com.vungle.ads.internal.protos.Sdk$SDKMetric;
import kotlin.Metadata;
import kotlin.coroutines.Continuation;
import kotlin.coroutines.jvm.internal.ContinuationImpl;
import kotlin.coroutines.jvm.internal.DebugMetadata;

/* JADX INFO: Access modifiers changed from: package-private */
@Metadata(k = 3, mv = {2, 1, 0}, xi = Sdk$SDKMetric.SDKMetricType.AD_SHOW_TO_VALIDATION_DURATION_MS_VALUE)
@DebugMetadata(c = "com.transsion.baselib.db.video.UGCCollectionVideoGroupDao$DefaultImpls", f = "UGCCollectionVideoGroupDao.kt", l = {44, 46, Sdk$SDKMetric.SDKMetricType.AD_OPTIONAL_DOWNLOAD_DURATION_MS_VALUE}, m = "addUgcVideoIds")
/* loaded from: classes5.dex */
public final class UGCCollectionVideoGroupDao$addUgcVideoIds$1 extends ContinuationImpl {
    Object L$0;
    Object L$1;
    Object L$2;
    int label;
    /* synthetic */ Object result;

    /* JADX INFO: Access modifiers changed from: package-private */
    public UGCCollectionVideoGroupDao$addUgcVideoIds$1(Continuation<? super UGCCollectionVideoGroupDao$addUgcVideoIds$1> continuation) {
        super(continuation);
    }

    @Override // kotlin.coroutines.jvm.internal.BaseContinuationImpl
    public final Object invokeSuspend(Object obj) {
        this.result = obj;
        this.label |= Integer.MIN_VALUE;
        return UGCCollectionVideoGroupDao.DefaultImpls.a(null, null, null, this);
    }
}
